app/Dockerfile.dev

```diff
@@ -0,0 +1,27 @@
+# Dockerfile
+
+# Use node alpine as it's a small node image
+FROM node:16-alpine3.14
+
+# Create the directory on the node image
+# where our Next.js app will live
+RUN mkdir -p /app
+
+# Set /app as the working directory
+WORKDIR /app
+
+# Copy package.json and package-lock.json
+# to the /app working directory
+COPY package*.json /app
+
+# Install dependencies in /app
+RUN yarn install
+
+# Copy the rest of our Next.js folder into /app
+COPY . /app
+
+# Ensure port 3000 is accessible to our system
+EXPOSE 3000
+
+# Run yarn dev, as we would via the command line
+CMD ["yarn", "dev"]
```

Comment:
Are you using this file? I don't see it in your docker compose.
If that's useful in your environment, happy to leave it.

Reply:
I am, I could also just keep it locally either way I'm okay.